package.json

```diff
@@ -40,43 +40,43 @@
   "peerDependencies": {
     "@remix-run/react": "^1.0.0",
     "@remix-run/server-runtime": "^1.0.0",
-    "i18next": "^21.3.3",
+    "i18next": "^21.3.3 || ^22.0.3",
     "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
-    "react-i18next": "^11.13.0"
+    "react-i18next": "^11.13.0 || ^12.0.0"
```

Comment:
The `react-i18next` version 12 upgraded the peer dependencies to support `i18next` version 22.